test/unit/pdf_application_form_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class PdfApplicationFormTest < ActiveSupport::TestCase
+  PDF_FIXTURE = './test/fixtures/pdfs/form_1.pdf'
+
+  test "it can get the fields as a hash" do
+    pdf_app = PdfApplicationForm.new(PDF_FIXTURE)
+
+    assert_equal "button", pdf_app.form_fields.first[:type]
+    assert_equal %w(0 1 2 3 Off), pdf_app.form_fields.first[:options]
+  end
+
+  test "#fill_form can fill out a form" do
+    pdf_app = PdfApplicationForm.new(PDF_FIXTURE)
+    fields = pdf_app.form_fields
+
+    attrs = {
+      pdf_app.form_fields.first[:name] => 1
+    }
+
+    Tempfile.create('temp_form') do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.